.github/workflows/ledger-suite-release.yml

```diff
@@ -0,0 +1,657 @@
+# Create GitHub releases for the ICP and ICRC ledger suite canisters.
+#
+# This workflow automatically triggers when a tag matching "ledger-suite-*" is pushed.
+# It parses the tag to extract release type and date, uses the tag's commit to download
+# artifacts from the IC mainnet build system, and creates a GitHub release with
+# verification instructions and change log.
+#
+# The release is created as a draft and can be published later. The Features part of the
+# release body is marked as "TODO" and should be filled in before publishing.
+#
+# Tag format: ledger-suite-{type}-{date}[.rcX]
+# - {type}: Either 'icrc' or 'icp'
+# - {date}: Date in YYYY-MM-DD format
+# - .rcX: Optional release candidate suffix (X = 1, 2, 3, etc.)
+#
+# Examples:
+#   Production: ledger-suite-icrc-2025-05-22, ledger-suite-icp-2025-05-22
+#   Release Candidates: ledger-suite-icrc-2025-05-22.rc1, ledger-suite-icp-2025-05-22.rc2
+#
+# Release Workflow:
+# 1. Pick a commit on the master branch (to ensure artifacts are available in CDN)
```

Comment:
Note that not every commit on the master branch has artifacts on the CDN. If a batch of commits in the merge queue is pushed to master only the last commit of the batch will trigger the `CI Main` workflow which will run the `Bazel Test All` job which will upload the artifacts ([example](https://github.com/dfinity/ic/commit/e861690a4fdfa8dbf0f897a3e9644141374a5efc)).

So maybe change the instructions to pick a commit on the master branch which ran the `Bazel Test All (push)` job.